serialization.md

```diff
@@ -327,7 +327,6 @@ Note: whitespace has been added to this example for clarity. Whitespace is OPTIO
         <ul>
           <li>
             <code>type</code> is usually set to <code>layers</code>.
-	    There is also a Windows-specific value <code>layers+base</code> that allows a base layer to be specified in a field of <code>rootfs</code> called <code>base_layer</code>.
```

Comment:
Are you leaving the “usually” language to allow extensions here to still be valid images?  Do you have a plan for validating unknown values?  It seems like a single option should be:

> `type` MUST be set to `layers`.

That gives you space to extend the type in the future.

But if there's only a single legal value, it seems like a waste of space.  Can't users look at the versioning information (the v1 in `application/vnd.oci.image.serialization.config.v1+json`?) and know that the rootfs type would be “layers” without having to specify it in the config?  Or do you expect future versions of the v1 serliazation spec to add support for additional types?